app/main.py

```diff
@@ -1,2 +1,54 @@
+from typing import Hashable, Any
+
+
+class Node:
+    def __init__(self, key: Hashable, value: Any) -> None:
+        self.key = key
+        self.value = value
+
+    def __repr__(self) -> str:
+        return f"{self.key}: {self.value}"
+
+
 class Dictionary:
-    pass
+    NEW_LEVEL = 2 / 3
```

Comment:
I would choose more descriptive name for this constant, something like "LOAD_FACTOR_THRESHOLD", for example.